### ISSUE TITLE
Split Data

### ISSUE THREAD

**Issue by Duffew** [Duffew/milestone-project-mildew-detection-in-cherry-leaves#7]
As a **data practitioner**, I want **to split the data** so that **I can train, test and validate the ML model**.

Acceptance criteria
- AC 1 - The images are split into folders with the following ratios:
- Train set = 70%
- Test set = 20%
- Validation set = 10%
